src/Content/Item.php

```diff
@@ -1025,8 +990,13 @@ public function initializePost(array $post): array
 		return $post;
 	}
 
-	public function finalizePost(array $post): array
+	public function finalizePost(array $post, bool $preview): array
 	{
+		if (!$preview) {
+			Attach::setPermissionFromBody($post);
+		} else {
+			$post['body'] = Attach::addAttachmentToBody($post['body'], $post['uid']);
+		}
```

Comment:
Please filp these outcomes so that the condition doesn't need the NOT operator.
```suggestion
		if ($preview) {
			$post['body'] = Attach::addAttachmentToBody($post['body'], $post['uid']);
		} else {
			Attach::setPermissionFromBody($post);
		}
```

Reply:
Done